src/services/redisSessionStorage.ts

```diff
@@ -320,6 +341,29 @@ export default class RedisSessionStorage extends RedisStorageUtils
     return right(some(user));
   }
 
+  /**
+   * {@inheritDoc}
+   */
+  public async getByFIMSToken(
```

Comment:
Ditto. Just thinking about a refactor to avoid creating another `getByXXXToken` in the future.

Reply:
https://github.com/pagopa/io-backend/pull/884#discussion_r845176331